package.json

```diff
@@ -13,6 +13,7 @@
         "prod:build": "yarn && yarn generate && nuxi generate",
         "prod:start": "nuxi start",
         "prepare:ci": "nuxi prepare",
+        "test": "yarn test:pinia --run && cypress open && yarn test:e2e",
```

Comment:
Let's move this to a separate PR since it's not related to the hamburger menu.

Reply:
I would pitch this is a refactoring improvement within the tests area I was touching 😎  
(but I can move it if I must 😿 )